src/test/java/core/basesyntax/service/impl/TransactionParserImplTest.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.TransactionParser;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class TransactionParserImplTest {
+    private static TransactionParser transactionParser;
+
+    @BeforeClass
+    public static void beforeClass() {
+        transactionParser = new TransactionParserImpl();
+    }
+
+    @Test
+    public void parse_Ok() {
+        List<FruitTransaction> actual = transactionParser.parseFile(List.of("skip Line",
+                "b,banana,20",
+                "b,apple,100",
+                "s,banana,100"));
+        List<FruitTransaction> expected = List.of(new FruitTransaction("b", "banana", 20),
+                new FruitTransaction("b", "apple", 100),
+                new FruitTransaction("s", "banana", 100));
+        Assert.assertEquals(actual, expected);
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void anotherTypeOperation_NotOk() {
```

Comment:
```suggestion
    public void anotherTypeOperation_NotOk() {
```
pattern for the naming of test methods should be like this `(name_of_the_method)_(special_state)_(ok/NotOk)`, fix naming for all tests